newrelic/resource_newrelic_api_access_key.go

```diff
@@ -333,3 +335,19 @@ func getAPIAccessKeyName(d *schema.ResourceData) string {
 
 	return name
 }
+
+// This method helps identify if the error caused is due to access forbidden to accounts
+// which are not entitled to creating Ingest License/Browser keys, based on the error message
+// (after an erroneous response is obtained from the API call made to create API keys).
+func refactorCreateAPIAccessKeyError(createErr error) error {
+	forbiddenAccessErrorClass := "data.apiAccessCreateKeys.errors"
+
+	forbiddenAccessErrorMessage := `Access forbidden.
```

Comment:
Is it possible to use the message returned by the API instead of using custom error message? 
resp
```
{
	"data": {
		"apiAccessCreateKeys": {
			"createdKeys": null,
			"errors": [{
				"accountId": 1234567,
				"id": null,
				"ingestErrorType": "FORBIDDEN",
				"ingestType": "LICENSE",
				"message": "You do not have permission to create this key.",
				"type": "INGEST"
			}]
		}
	}
}

```

Reply:
@NSSPKrishna I did want to have the message returned by the API instead of a custom message - but the method in client-go to create API keys, `CreateAPIAccessKeysWithContext` does not return the response of the API, and returns only the key(s) created and and error if any - which is why I think, without a change from client-go, we cannot have this method in Terraform obtain the response sent by the API